common/djangoapps/student/views.py

```diff
@@ -707,7 +707,11 @@ def dashboard(request):
     elif not user.is_active:
         banner_account_activation_message = render_to_string(
             'registration/activate_account_notice.html',
-            {'email': user.email}
+            {
+                'email': user.email,
+                'platform_name': platform_name,
+                'activation_email_support_link': activation_email_support_link,
+            }
```

Comment:
Nit: Since this `dict` seems to be the same as the one above it, can we possibly factor it out?